css/viewer/annotationContainer.css

```diff
@@ -47,3 +47,59 @@
 #channel-names-overlay-container .channel-label-overlay.visible {
     display: inline-block;
 }
+
+
+/* 
+CSS for text annotation tool 
+*/
+
+.foreign-object {
+    text-align: left;
+    overflow: visible;
+}
+
+.foreign-object-div {
+    display: inline-block;
+    border: 100px solid red;
+    /* position: absolute; */
```

Comment:
Remove this rule if it's not needed. Or if there's a reason why you left it as commented (instead of removing it), comment it.

Reply:
Removed